provider/build.gradle

```diff
@@ -61,6 +61,13 @@ android {
             }
         }
     }
+
+    publishing {
+        singleVariant("release") {
+            withSourcesJar()
+            withJavadocJar()
+        }
+    }
```

Comment:
Following example in https://developer.android.com/build/publish-library/configure-pub-variants#single-pub-var